src/engine/TrackBrowser.cpp

```diff
@@ -26,6 +26,9 @@ void TrackBrowser::FindCustomTracks() {
             TrackInfo info;
             info.Path = dir;
             TrackEditor::LoadTrackInfo(info, archive, sceneFile);
+            if (info.ResourceName.empty()) {
+                info.ResourceName = std::string("mods:") + name;
```

Comment:
Not sure I like this.

This should be set by the user when they create the scene.json file. It shouldn't be empty.

Reply:
I took a closer look. I'm thinking this should also have something like

`printf("[TrackBrowser] Track has invalid resource name; expected format: author:mod_name");`
or "Track missing ResourceName; expected format author:mod_name\n  Please set in HM64 labs"
Perhaps there should be a verification function or something